js/bitfinex2.js

```diff
@@ -435,19 +435,55 @@ module.exports = class bitfinex2 extends bitfinex {
         throw new NotSupported (this.id + ' withdraw not implemented yet');
     }
 
-    async fetchMyTrades (symbol = undefined, since = undefined, limit = 25, params = {}) {
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = 1000, params = {}) {
         await this.loadMarkets ();
         let market = this.market (symbol);
         let request = {
             'symbol': market['id'],
             'limit': limit,
-            'end': this.seconds (),
+            'end': this.milliseconds (),
         };
         if (since !== undefined)
-            request['start'] = parseInt (since / 1000);
+            request['start'] = since.getTime ();
         let response = await this.privatePostAuthRTradesSymbolHist (this.extend (request, params));
-        // return this.parseTrades (response, market, since, limit); // not implemented yet for bitfinex v2
-        return response;
+        return this.parseMyTrades (response, market);
```

Comment:
There's no parseMyTrades/parseMyTrade in the call chain, all trades are parsed with `fetchTrades` / `parseTrade`, including the public and the private ones.

Reply:
Ok, I've moved the logic to the parseTrade method.